next-sitemap.config.js

```diff
@@ -0,0 +1,8 @@
+/** @type {import('next-sitemap').IConfig} */
+// Default code you can customize according to your requirements.
+module.exports = {
+    siteUrl: process.env.SITE_URL || 'https://razakhawaja.me',
```

Comment:
Can we pick this from a public json?

Reply:
The easiest and recommended way to do this is to have a .env.local file from where we pick the site url and add the env variable to vercel. I can change the hardcoded url to localhost or simply remove it.

Reply:
Glanced over the env variable and missed it. Changing the hardcoded url to localhost sounds great